smarts/core/vehicle.py

```diff
@@ -531,9 +531,9 @@ def _meta_create_sensor_functions(self):
         for sensor_name in sensor_names:
 
             def attach_sensor(self, sensor, sensor_name=sensor_name):
-                assert (
-                    getattr(self, f"_{sensor_name}", None) is None
-                ), f"{sensor_name} already added"
+                # assert (
```

Comment:
This comment out will be reverted but I found that somehow vehicles with sensors were having their sensor replaced.

Run `python examples/egoless.py scenarios/minicity`

Reply:
This should be reverted.